src/foam/u2/DateTimeView.js

```diff
@@ -46,14 +46,22 @@ foam.CLASS({
       var slot    = this.attrSlot(); //null, this.onKey ? 'input' : null);
 
       function updateSlot() {
-        var date = self.data;
-        if ( ! date ) {
+        if ( ! self.data ) {
           slot.set('');
         } else {
-          if ( ! date.toISOString )
-            date = new Date(date);
-
-          slot.set(date.toISOString().substring(0,16));
+          // Convert date with timezone into ISO format
```

Comment:
Does this work: new Date(self.data).toISOString();

Reply:
This is what it was.